project/properties/templates/properties/singleProperty.html

```diff
@@ -1,24 +1,34 @@
 {% extends "base.html" %}
 
 {% block css %}
-<link rel="stylesheet" href="/static/properties/css/properties.css">
+<link rel="stylesheet" href="/static/properties/css/singleProperty.css">
 {% endblock %}
 
 
 {% block body %}
 	<div class="row titlesArea">
 		<div class="col s4 titles">
+			<h3>Images</h3>
+		</div>
+		<div class="col s4 titles">
 			<h3>Google Map</h3>
 		</div>
-		<div class="col s8 titles">
+		<div class="col s4 titles">
 			<h3>Property Facts</h3>
 		</div>
 	</div>
 
 	<div class="row property">
-		<div class="col s4 propertyInfoArea googleMaps" data-lat="{{lat}}" data-long="{{long}}">
+		<div class="col s4 propertyInfoArea slideShow">
+			<img class = "slideShowImage" src="/static/properties/images/picture1.jpg">
+			<img class = "slideShowImage" src="/static/properties/images/picture2.jpg">
+			<img class = "slideShowImage" src="/static/properties/images/picture3.jpg">
+			<button class="slideShowButton" onclick="plusDivs(-1)">&#10094;</button>
+			<button class="slideShowButton" onclick="plusDivs(+1)">&#10095;</button>
```

Comment:
<img> tags are all created which might mean we will do this via templating. The javascript is what controls which image is displayed and not all of them at once.